### PR TITLE
Implementation of Google Sign In 

### PR DESCRIPTION
**Pull request by MarcelTorne**

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ahmedc3ca) [inline .idea/gradle.xml:1]: Do you think it's necessary to include the .idea/gradle.xml on the remote version?

**Review by ahmedc3ca — APPROVED** [review]
Everything Looks great, good job! Only remark is about the inclusion of the .idea/gradle.xml file on the remote

**Comment by aman3014** [comment]
Looks good! Just don't forget to reformat the code using Android Studio (on Windows it's Ctrl+Alt+L) 😄

File: .idea/gradle.xml (hunk 1 above)
Comment (MarcelTorne) [reply .idea/gradle.xml:1]: No, indeed, I am going to remove it.

**Review by aman3014 — APPROVED** [review]

**Pull request merged**